catt/controllers.py

```diff
@@ -70,6 +70,8 @@ def setup_cast(device_name, video_url=None, prep=None, controller=None):
                        controller is bypassed, and the one specified here is
                        returned instead.
     :type controller: str
+    :param ytdl_option: YoutubeDL options
```

Comment:
This should be a bit more descriptive, something like `Pairs of options to be passed to YoutubeDL.`

Reply:
Updating it as below if it's ok
```python 
ytdl_options: Pairs of options to be passed to YoutubeDL. For the available options to please refer to
                        https://github.com/rg3/youtube-dl/blob/master/youtube_dl/YoutubeDL.py#L138-L317
```

Reply:
Ah, yes, that's great. Can you wrap the sentence a bit more so it looks tidier? Obviously the URL can't wrap, but that's okay.

Reply:
Divided it to three lines, it's better now.